app/scripts/controllers/accounting/SearchTransactionController.js

```diff
@@ -116,10 +118,15 @@
                 scope.formData.savingsaccountId = null;
             };
 
-            scope.searchTransaction = function () {
-                scope.displayResults = true;
-                scope.transactions = paginatorService.paginate(fetchFunction, 14);
-                scope.isCollapsed = false;
+            scope.submit = function () {
```

Comment:
keep the name of the function as searchTransaction. Its not required to change the name of the method if you change button type to submit